app/src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="com.burhanrashid52.photoediting">
```

Comment:
Change made by Upgrade Assistant: Move package from Android manifest to build files